src/main/java/com/esri/samples/portal/integrated_windows_authentication/README.md

```diff
@@ -0,0 +1,48 @@
+# Integrated windows authentication
+
+Use Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).
+
+![](IntegratedWindowsAuthentication.png)
+
+## Use case
+
+Integrated Windows Authentication, which is built into Microsoft Internet Information Server (IIS), works well for intranet applications, but isn't always practical for internet apps.
+
+## How to use the sample
+
+1. Enter the URL to your IWA-secured portal into the text field.
+2. Select either the `Search Public Portal` button (which will search for portals on www.arcgis.com) or `Search IWA Secured Portal` (which will search your IWA-secured portal), to search for web maps stored on the portal.
+3. If you chose to search a secured portal, you will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password.
+4. If you authenticate successfully, portal item results will display in the list.
+5. Select a web map item to display it in the map view.
+
+## How it works
+
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username@domain and password) if a secure resource is encountered.
+2. When a search for portal items is performed against an IWA-secured portal, the challenge handler creates a `UserCredential` object from the information entered by the user.
+3. If the user authenticates, the search returns a list of web maps (`ArcGISPortalItem`) and the user can select one to display as a `Map`.
+
+## Relevant API
+
+* `AuthenticationChallenge`
+* `AuthenticationChallengeResponse`
+* `AuthenticationManager`
+* `AuthenticationManager.setAuthenticationChallengeHandler`
+* `Portal`
+* `UserCredential`
+
+## About the data
+
```

Comment:
This section isn't so much about the data and is more so how to use the app. The info after the very first sentence can be incorporated in the How to use this sample section. The first sentence already here should suffice for this section.

Reply:
That's very true. I've tried what you suggested, but then figured it's just a repetition of what's already in `how to use`, so I've removed this section alltogether. If anything, we could describe the contents of the public portal because that is what will be seen by all users?